.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -58,20 +58,36 @@ jobs:
             experimental/regression_suite
 
       # Out of tree tests
-      - name: Checking out external TestSuite repository
+      - name: Install git lfs
+        run: |
+          sudo apt-get install git-lfs
+          git lfs install
+      - name: Check out external TestSuite repository
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: 46ab63b271fddd78e071c0590e9a073f6282b005
+          ref: 65cdac8e1dfab53892337009a9704162e1cf8bfb
           path: SHARK-TestSuite
           submodules: false
-      - name: Installing external TestSuite Python requirements
+          lfs: true
+      - name: Install external TestSuite Python requirements
         run: |
           source ${VENV_DIR}/bin/activate
           python -m pip install -r SHARK-TestSuite/iree_tests/requirements.txt
-      - name: Run external TestSuite tests
-        env:
-          IREE_TEST_CONFIG_FILES: experimental/regression_suite/external_test_suite/config_cpu_llvm_sync.json
+      - name: Run external tests - ONNX test suite
```

Comment:
We can also use other pytest configurations (conftest.py) or test runners. The `iree_tests` setup is just one flavor of testing that I've designed to work with stable .mlir files, inputs, and outputs. Other tests using Python scripts (e.g. running PyTorch and Turbine) _could_ be exercised here, but we want to avoid depending on too many other projects here in IREE itself.